docs/product-oracle-weblogic-installation-and-configuration-guide/src/main/asciidoc/kie-server-wls-environment-set-proc.adoc

```diff
@@ -49,39 +45,62 @@ ifdef::DM[]
 endif::DM[]
 
 ifdef::BA[]
+|`kie.services.jms.queues.response`
+|`jms/KIE.RESPONSE.ALL`
+|The JNDI name of JMS queue for responses used by the {CENTRAL} and {KIE_SERVER}.
+
+|`org.uberfire.domain`
+|`OracleDefaultLoginConfiguration`
+|JAAS `LoginContext` domain used to authenticate users when using JMS.
+
 |`org.kie.server.persistence.ds`
-|Set according to database type and configuration.
-|Datasource JNDI name.
+|`jdbc/jbpm`
+|Data source JNDI name for {KIE_SERVER}.
 
 |`org.kie.server.persistence.tm`
 |`org.hibernate.service.jta.platform.internal.WeblogicJtaPlatform`
 |Transaction manager platform for setting Hibernate properties.
 
 |`org.kie.server.persistence.dialect`
-|Set according to database type and configuration.
-|Specifies the Hibernate dialect to be used.
+|Example: `org.hibernate.dialect.H2Dialect`
+|Specifies the Hibernate dialect to be used. Set according to data source.
+
+|`org.kie.executor.jms.queue`
+|`jms/KIE.EXECUTOR`
```

Comment:
it should default to jms/KIE.SERVER.EXECUTOR

Reply:
@mswiderski, does this only apply to JMS queues, or should I also make the same change for JMS connection factories (change from `jms/cf/KIE.EXECUTOR` to `jms/cf/KIE.SERVER.EXECUTOR`) and for JMS activation specifications (change from `jms/activation/KIE.EXECUTOR` to `jms/activation/KIE.SERVER.EXECUTOR`).